### DISCUSSION TITLE
what is the @/styles path?

### DISCUSSION THREAD

**Discussion by hp8wvvvgnj6asjm7** [vercel/next.js#52164] [App Router]
`import '@/styles/globals.css'`

https://nextjs.org/docs/pages/building-your-application/styling/tailwind-css

**Comment by icyJoseph** [comment]
It is called absolute path. It means that they want to load a module from `./src/styles`.

You can tell TypeScript, or JavaScript compilers, how to resolve certain paths. Here's an article that does this on a generic TS project: https://dev.to/seanyasno/how-to-add-absolute-paths-to-your-typescript-project-b62

In Next.js, you can find this in `tsconfig.json` ( or `jsconfig.json` if working with JS):

```json
{
  "compilerOptions": {
    "target": "es5",
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "incremental": true,
    "plugins": [
      {
        "name": "next"
      }
    ],
    "paths": {
      "@/*": [
        "./src/*"
      ]
    }
  },
  "include": [
    "next-env.d.ts",
    "**/*.ts",
    "**/*.tsx",
    ".next/types/**/*.ts",
    "rsc-fetch-view/interceptors.js"
  ],
  "exclude": [
    "node_modules"
  ]
}
```

You see the paths key, which uses the `@/` to map to `./src`

And here the `jsconfig.json` alternative:

```json
{
  "compilerOptions": {
    "paths": {
      "@/*": ["./src/*"]
    }
  }
}
```
